src/controllers/commentController.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable class-methods-use-this */
+import Response from '../utils/response';
+import CommentService from '../repositories/commentRepository';
+import UserService from '../repositories/userRepository';
+import RequestService from '../repositories/requestRepository';
+
+class CommentController {
+  /**
+   * Creates a new comment.
+   * @param {object} req request
+   * @param {object} res response
+   * @param {object} next next
+   * @returns {object} response object
+   */
+  static async addComment(req, res, next) {
+    try {
+      const { comment } = req.body;
+      const id = Number(req.params.id);
```

Comment:
Hi @KarangwaEmmy this is id of what? could you make your variable naming more descriptive. Thank you.